.textlintrc.json

```diff
@@ -83,10 +83,12 @@
         "YouTube",
         "Zend",
         "Zipkin",
+        [".net", ".NET"],
         ["3rd[- ]party", "third-party"],
-        ["bug[- ]fix(es)?", "bugfix$1"],
```

Comment:
Do you want to add the reverse rule then?

Reply:
Yeah, we should.

Reply:
Eh, I don't know if we should add the reverse rule. Both are fine. I don't feel strongly about this though.

Reply:
I think that we should opt for consistency, but we can add the reverse rule in a followup PR.